app/misc/authenticate.ts

```diff
@@ -84,13 +85,33 @@ function ModalSignIn(callback) {
 // previously we were trying to use same object again and again which was cauing issues
 function getCredentials(){
   //Note: might need to update with Oauth
-  return Git.Cred.userpassPlaintextNew(getUsernameTemp(), getPasswordTemp());
+  return Git.Cred.userpassPlaintextNew(getAccessToken(), getPasswordTemp());
 }
 
 
 function loginWithSaved(callback) {
-    document.getElementById("username").value = getUsername();
-    document.getElementById("password").value = getPassword(); //get decrypted username n password
+  // Get the saved access token from the file system
+  var accessToken = getToken();
+
+  // Store the token in memory
+  encryptTemp(accessToken);
```

Comment:
bump

Reply:
The reason we use a function outside of the authenticate file is really only because that is the pattern that exists in the project prior to these changes. This is how encryptedUsername and encryptedPassword were set. I will can change it though.